components/Modal/index.d.ts

```diff
@@ -1,7 +1,4 @@
-import { Component, ComponentType, ReactNode, TouchEvent, MouseEventHandler } from 'react';
-import HeaderText from '../Card/HeaderText';
-import Title from '../Card/Title';
-import Footer from '../Card/Footer';
+import { ComponentType, ReactNode, TouchEvent, MouseEventHandler } from 'react';
```

Comment:
```suggestion
import { Component, ComponentType, ReactNode, TouchEvent, MouseEventHandler } from 'react';
```

Reply:
React.Component doesn't work with subcomponents, Because it is not possible to type in internal components.

Reply:
Why? Can you show an attachment with the problem?